lib/explorer/backend/lazy_series.ex

```diff
@@ -40,14 +40,20 @@ defmodule Explorer.Backend.LazySeries do
     median: 1,
     var: 1,
     std: 1,
-    quantile: 2
+    quantile: 2,
+    # Maybe aggregations
```

Comment:
Nitpick: I think they are always aggregations. Maybe the name aggregation is confusing here but the idea is that they only make sense when considering multiple rows, not a single one.

```suggestion
```